### ISSUE TITLE
[Feat] cluster support

### ISSUE THREAD

**Issue by Coachonko** [patrickpissurno/vredis#17]
Cluster support is missing right now. Should be implemented

**Comment by patrickpissurno** [comment]
Agreed! I don't have the time to work on that right now, but feel free to open up a PR and I'll make sure to find some time to review it and get it merged quickly
